src/executor/plan_executor.cpp

```diff
@@ -101,6 +100,7 @@ static void InterpretPlan(
   status = executor_tree->Init();
   if (status != true) {
     result.m_result = ResultType::FAILURE;
+    result.m_error_message = "Executor tree init failed";
```

Comment:
Can you make this a little more descriptive? Something like "_Failed initialization of query execution tree_"?